community/open-meetings.mdx

```diff
@@ -7,6 +7,38 @@ These are dedicated sync meetings for specific products, as well as open plannin
 
 ## Regular meetings
 
+<MeetingInfo title="Office Hour"
+             schedule="Every Friday effective 16. Feb 2024 until 31. Dec 2024 from 01:05 pm to 02:00 pm CET (UTC +1)"
+             description="Open hour meeting for all interests. The goal of the meeting is to inform and share information about different topics."
+             contact="stephan.bauer@catena-x.net"
+             sessionLink="https://teams.microsoft.com/l/meetup-join/19%3ameeting_MDFiNDJjMmQtNjFkYi00ODdjLTk2NDgtZGMwNTRmYzg3NzM0%40thread.v2/0?context=%7b%22Tid%22%3a%221ad22c6d-2f08-4f05-a0ba-e17f6ce88380%22%2c%22Oid%22%3a%22a8b7a5ee-66ff-4695-afa2-08f893d8aaf6%22%7d"
+             meetingLink="/meetings/office-hour.ics"
+/>
+
+<MeetingInfo title="NewJoiner - Office Hour"
+             schedule="The second Friday of every month effective 8. Mar 2024 until 31. Dec 2024 from 02:05 pm to 03:00 pm CET (UTC +1)"
+             description="Open hour meeting for all new joiner. The goal of the meeting is to provide an easy entry on all topics. The meeting will only take place if specific questions have been submitted."
+             contact="stephan.bauer@catena-x.net"
+             sessionLink="https://teams.microsoft.com/l/meetup-join/19%3ameeting_ZjAzYzVjZGEtZjEwNC00NDI5LWEwODEtN2RhNmI0NDEzNTI2%40thread.v2/0?context=%7b%22Tid%22%3a%221ad22c6d-2f08-4f05-a0ba-e17f6ce88380%22%2c%22Oid%22%3a%22a8b7a5ee-66ff-4695-afa2-08f893d8aaf6%22%7d"
+             meetingLink="/meetings/newjoiner-office-hour.ics"
+/>
+
+<MeetingInfo title="Committer - Office Hour"
```

Comment:
I'm not sure if "Committer Office Hour" is the right name for this meeting. It is meant as a meeting for Committers to discuss Eclipse Tractus-X and/or Committer related topics. I'd name it more generic, e.g. "Committer Meeting".